draft-ietf-masque-h3-datagram.md

```diff
@@ -570,6 +570,20 @@ all cases, the maximum permitted value of the H3_DATAGRAM SETTINGS parameter is
 1.
 
 
+## Note About Draft Versions
+
+\[\[RFC editor: please remove this section before publication.]]
+
+Some revisions of this draft specification use a different value for the
+H3_DATAGRAM Settings Parameter. This allows new draft revisions to make
+incompatible changes. Implementations MAY support multiple draft versions at
+the same time. In that scenario, endpoints MUST send multiple values for
+H3_DATAGRAM. Once an endpoint has sent and received SETTINGS, it MUST compute
+the intersection of the values it has sent and received, and then it MUST
+select and use the most recent draft version from the intersection set. This
+ensures that both endpoints negotiate the same draft version.
```

Comment:
This is a good change.

One thought, since `H3_DATAGRAM` is just a logical identifier we could have encoded the draft version into it and made it easier to talk about. But I think that ship has probably sailed.

Reply:
Agreed. That's what I plan on doing in our code. I think using `H3_DATAGRAM` in the draft and when it's ambiguous in email or code using `H3_DATAGRAM-03` seems fine.

Reply:
WFM